README.md

```diff
@@ -1240,6 +1240,176 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='blank-line-after-multiline-switch-case'></a>(<a href='#blank-line-after-multiline-switch-case'>link</a>) **Insert a blank line following a switch case with a multi-line body.** Spacing within an individual switch statement should be consistent. If any case has a multi-line body then all cases should include a trailing blank line. The last switch case doesn't need a blank line, since it is already followed by a closing brace. [![SwiftFormat: blankLineAfterMultilineSwitchCase](https://img.shields.io/badge/SwiftFormat-blankLineAfterMultilineSwitchCase-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#blankLineAfterMultilineSwitchCase) [![SwiftFormat: consistentSwitchStatementSpacing](https://img.shields.io/badge/SwiftFormat-consistentSwitchStatementSpacing-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#consistentSwitchStatementSpacing)
```

Comment:
Asking to learn... how does this proposal cover (or not cover) situations where the case itself is multiple lines? I think adding examples to show how this is handled could be nice.

```swift
switch myValue {
case .one:
  doSomething()
case .two, .three, .four, .five, .six, .seven, .eight, .nine, .ten,
    .eleven, .twelve:
  doSomethingElse()
}

Reply:
The rule implementation counts the number of lines between the `:` following the `case` keyword, and the next case / closing brace. The prose of the rule tries to address this by saying referring to "switch cases with a multi-line body" as opposed to "multi-line switch cases".